suites/selftest/ts/tools/package.xml

```diff
@@ -529,5 +529,38 @@
                 </run>
             </session>
         </run>
+
+        <run name="default_args" template="default_arg_test_s">
+            <session>
+                <arg name="test_arg1"/>
+                <arg name="test_arg2"/>
+                <run>
+                    <script name="default_args"/>
+                </run>
+                <run>
+                    <script name="default_args"/>
+                    <arg name="test_arg2">
+                        <value>99</value>
+                    </arg>
+                </run>
+                <run template="default_arg_test">
+                    <script/>
```

Comment:
It looks rather strange. IMHO it should work without it.

Reply:
If I use <run template="default_arg_test"/> instead I get: Empty 'run' item

Reply:
It must be fixed.